doc/source/ray-air/examples/torch_image_batch_pretrained.py

```diff
@@ -36,4 +36,4 @@ def preprocess(batch: np.ndarray) -> pd.DataFrame:
 ckpt = TorchCheckpoint.from_model(model=model, preprocessor=preprocessor)
 
 predictor = BatchPredictor.from_checkpoint(ckpt, TorchPredictor)
-predictor.predict(dataset, batch_size=80)
+predictor.predict(dataset)
```

Comment:
This is manual change after resolving conflict, please double check @clarkzinzow.